wagtail/admin/templatetags/avatar.py

```diff
@@ -0,0 +1,18 @@
+from __future__ import absolute_import, unicode_literals
```

Comment:
I suggest we move this tag into `wagtailadmin_tags`, to avoid the possibility of clashing with a tag library with the name `avatar` within a user's project code. (I'm pretty sure gravatar.py was only a separate module in the first place due to casual copy-and-pasting of code found on the internet...)